quickstarts/karaf-camel-rhosak-template.json

```diff
@@ -0,0 +1,364 @@
+{
+  "apiVersion": "v1",
+  "kind": "Template",
+  "metadata": {
+    "annotations": {
+      "openshift.io/display-name": "Red Hat Fuse 7.9 Camel RHOSAK with Karaf",
+      "openshift.io/provider-display-name": "Red Hat, Inc.",
+      "description": "A simple Camel route for Red Hat Openshift streams for Apache Kafka. This quickstart shows a simple Apache Camel application that connects with Red Hat OpenShift Streams for Apache Kafka.",
```

Comment:
strams is wirtten oen time with a capital letter and one time without.

Reply:
Correcting in commit to capital letter to match quickstart readme